torchao/_models/llama/eval.py

```diff
@@ -60,17 +60,18 @@ def run_evaluation(
 
     if quantization:
         if "int8wo" in quantization:
-            quantize(model, int8wo())
+            quantize(model, int8_weight_only())
```

Comment:
does this need to be compatible with torch 2.3 and below? if so we could define similar helpers: https://github.com/pytorch/ao/blob/bc8599fb96588cf287c311fa38669288b1de2f3f/test/integration/test_integration.py#L99-L118

Reply:
i think its mostly for our own testing